json.d.ts

```diff
@@ -1,4 +0,0 @@
-declare module '*.json' {
```

Comment:
I thought this seemed dumb but I think I fixed the issue I was having and just never got rid of it 👍

Reply:
I thought so, all good!